src/cmd.rs

```diff
@@ -33,7 +33,7 @@ pub struct TopLevel {
 
 #[derive(FromArgs, PartialEq, Debug)]
 #[argh(subcommand)]
-/// hikalium's ChromiumOS dev commands
+/// lium's ChromiumOS dev commands
```

Comment:
maybe we should keep this :)

Reply:
I understand the sentiment but I think it would cause confusion for external chromiumOS contributors :\